README.md

```diff
@@ -8,7 +8,7 @@
 ## Deployment
 To bundle into a single platform specyfic executable, run:
     
-    pyinstaller -F -w app.py
+    pyinstaller -F -w src/app.py
```

Comment:
For now, it is ok, but there is compile.py which will be used as the python wrapper for this command